[2025-03-16T03:00:03.076+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-16T03:00:03.089+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.bronze_layer_group.check_minio_connection scheduled__2025-03-16T02:00:00+00:00 [queued]>
[2025-03-16T03:00:03.096+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.bronze_layer_group.check_minio_connection scheduled__2025-03-16T02:00:00+00:00 [queued]>
[2025-03-16T03:00:03.096+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2025-03-16T03:00:03.105+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): bronze_layer_group.check_minio_connection> on 2025-03-16 02:00:00+00:00
[2025-03-16T03:00:03.113+0000] {standard_task_runner.py:72} INFO - Started process 832 to run task
[2025-03-16T03:00:03.117+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'bronze_layer_group.check_minio_connection', 'scheduled__2025-03-16T02:00:00+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmpqoy_td9l']
[2025-03-16T03:00:03.118+0000] {standard_task_runner.py:105} INFO - Job 63: Subtask bronze_layer_group.check_minio_connection
[2025-03-16T03:00:03.164+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.bronze_layer_group.check_minio_connection scheduled__2025-03-16T02:00:00+00:00 [running]> on host 3174348cc1e9
[2025-03-16T03:00:03.225+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='bronze_layer_group.check_minio_connection' AIRFLOW_CTX_EXECUTION_DATE='2025-03-16T02:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-16T02:00:00+00:00'
[2025-03-16T03:00:03.226+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-16T03:00:03.230+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-03-16T03:00:03.233+0000] {base.py:84} INFO - Retrieving connection 'minio_conn'
[2025-03-16T03:00:03.233+0000] {warnings.py:109} WARNING - <string>:9: UserWarning: AWS Connection (conn_id='minio_conn', conn_type='s3') expected connection type 'aws', got 's3'. This connection might not work correctly. Please use Amazon Web Services Connection type.

[2025-03-16T03:00:03.234+0000] {connection_wrapper.py:325} INFO - AWS Connection (conn_id='minio_conn', conn_type='s3') credentials retrieved from login and password.
[2025-03-16T03:00:03.761+0000] {python.py:240} INFO - Done. Returned value was: True
[2025-03-16T03:00:03.786+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-16T03:00:03.786+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=bronze_layer_group.check_minio_connection, run_id=scheduled__2025-03-16T02:00:00+00:00, execution_date=20250316T020000, start_date=20250316T030003, end_date=20250316T030003
[2025-03-16T03:00:03.820+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-16T03:00:03.842+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-16T03:00:03.843+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-03-16T03:48:23.495+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-16T03:48:23.511+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.bronze_layer_group.check_minio_connection scheduled__2025-03-16T02:00:00+00:00 [queued]>
[2025-03-16T03:48:23.519+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.bronze_layer_group.check_minio_connection scheduled__2025-03-16T02:00:00+00:00 [queued]>
[2025-03-16T03:48:23.519+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2025-03-16T03:48:23.530+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): bronze_layer_group.check_minio_connection> on 2025-03-16 02:00:00+00:00
[2025-03-16T03:48:23.537+0000] {standard_task_runner.py:72} INFO - Started process 137 to run task
[2025-03-16T03:48:23.542+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'bronze_layer_group.check_minio_connection', 'scheduled__2025-03-16T02:00:00+00:00', '--job-id', '102', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmpbm9tq12j']
[2025-03-16T03:48:23.544+0000] {standard_task_runner.py:105} INFO - Job 102: Subtask bronze_layer_group.check_minio_connection
[2025-03-16T03:48:23.598+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.bronze_layer_group.check_minio_connection scheduled__2025-03-16T02:00:00+00:00 [running]> on host c891f3a5df31
[2025-03-16T03:48:23.673+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='bronze_layer_group.check_minio_connection' AIRFLOW_CTX_EXECUTION_DATE='2025-03-16T02:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-16T02:00:00+00:00'
[2025-03-16T03:48:23.674+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-16T03:48:23.678+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-03-16T03:48:23.683+0000] {base.py:84} INFO - Retrieving connection 'minio_conn'
[2025-03-16T03:48:23.684+0000] {warnings.py:109} WARNING - <string>:9: UserWarning: AWS Connection (conn_id='minio_conn', conn_type='s3') expected connection type 'aws', got 's3'. This connection might not work correctly. Please use Amazon Web Services Connection type.

[2025-03-16T03:48:23.684+0000] {connection_wrapper.py:325} INFO - AWS Connection (conn_id='minio_conn', conn_type='s3') credentials retrieved from login and password.
[2025-03-16T03:48:24.267+0000] {python.py:240} INFO - Done. Returned value was: True
[2025-03-16T03:48:24.288+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-16T03:48:24.289+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=bronze_layer_group.check_minio_connection, run_id=scheduled__2025-03-16T02:00:00+00:00, execution_date=20250316T020000, start_date=20250316T034823, end_date=20250316T034824
[2025-03-16T03:48:24.342+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-16T03:48:24.362+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-16T03:48:24.364+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
